bin/show_commit_history.rb

```diff
@@ -11,10 +11,14 @@
   opt :to,   "The commit log 'to' ref" ,  :type => :string,  :required => true
   opt :skip, "The repos to skip",         :type => :strings, :default => ["manageiq-documentation"]
 
+  opt :pr_titles,  "Whether to lookup PR details", :type => :boolean, :default => false
```

Comment:
Should this option be `type` with values of `commit` and `pr` (defaulting to `commit`)?

Reply:
I like the idea of a "filter".  I'll think about that

Reply:
Decided on `:display`